src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs

```diff
@@ -60,7 +60,7 @@ public static ValueTask<QuicConnection> ConnectAsync(QuicClientConnectionOptions
     {
         if (!IsSupported)
         {
-            throw new PlatformNotSupportedException(SR.Format(SR.SystemNetQuic_PlatformNotSupported, MsQuicApi.NotSupportedReason));
+            throw new PlatformNotSupportedException(SR.Format(SR.SystemNetQuic_PlatformNotSupported, MsQuicApi.NotSupportedReason ?? "General loading failure."));
```

Comment:
Why this change? Can `NotSupportedReason` be null here?

Reply:
Given that it's nullable string, I thought that should be fine to provide something like `General loading failure`, instead of having it `emtpy`.

Reply:
But can it be empty? Are there cases when it's actually null?

Reply:
Looks like it can, if I'm not missing something.

https://github.com/dotnet/runtime/blob/81441a1b1abc46309ce472539aac83af7f18cd39/src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/MsQuicApi.cs#L136-L142